src/lib/Signatures.sol

```diff
@@ -18,6 +18,10 @@ abstract contract Signatures {
                      SIGNATURE VERIFICATION HELPERS
     //////////////////////////////////////////////////////////////*/
 
+    function _verifySigPlain(bytes32 digest, address signer, bytes memory sig) internal view returns (bool is_valid) {
```

Comment:
do we need this helper? it's easier to understand whats going on if we just call it inline in the verifyFidSignature method.

Reply:
I thought I'd add it if the same sort of external view would be used somewhere else.

Removed.